src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -16,14 +16,14 @@
    :right-icon          :i/external})
 
 (defn- action-send
-  [send-params]
+  [send-params entry-point]
   {:icon                :i/send
    :accessibility-label :send
    :label               (i18n/label :t/send)
    :on-press            (fn []
                           (rf/dispatch [:hide-bottom-sheet])
                           (rf/dispatch [:wallet/clean-send-data])
-                          (rf/dispatch [:wallet/set-token-to-send send-params]))})
+                          (rf/dispatch [:wallet/set-token-to-send send-params entry-point]))})
```

Comment:
An `entry-point` parameter has been added, this will define where we are pressing the token and what information we should add or remove while navigating forward and backward respectively.

Currently only exists `:wallet-home` (added when we send from the wallet home)

Reply:
Could we use the current screen name as the key instead of using arbitrary keys like :wallet-home? If not, can we create constants for these keys to make the code more maintainable? This way, we'll have a single source of truth for the possible values of the entry-point variable.

Reply:
@briansztamfater  Sure!

It's a great suggestion, I'll apply it!

Reply:
@briansztamfater The wallet home component is not mounted on the screen navigation hierarchy, so that it doesn't have a screen name. Creating the keyword `:screen/wallet-home` for this purpose is misleading, so better to keep it as is, but for other valid entry-points I'll apply your suggestion.

Reply:
AFAIK, you can get the screen name from `:view-id` key in app-db. From what I could find by debugging with re-frisk, "wallet home" screen name is `:wallet-stack` (as it is the root screen of wallet tab stack).